procurement/static/css/_forms.scss

```diff
@@ -0,0 +1,15 @@
+.form-group {
+    margin-right: 1rem;
+    margin-top: 1rem;
+    label {
+        margin-bottom: 0.5em;
+    }
+}
+
+.btn-dropdown {
```

Comment:
Ah clever, making use of the browser’s default `<summary>` styling. I feel like, since this will only have the desired effect on a `<summary>` element, maybe it should use that, instead of a generic class like `.btn-dropdown`. So, `summary.btn` instead, maybe?

Reply:
I’d probably also add a comment to that line, explaining that `display: list-item` will reintroduce the browser’s default "toggle" triangle on the `summary` element. Because otherwise, seeing `display: list-item` on a button, with no other context, is a bit of a head scratcher!

Reply:
One for @lucascumsille